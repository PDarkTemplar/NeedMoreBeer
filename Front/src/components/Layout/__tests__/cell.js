// @flow
import { shallow } from 'enzyme';

import React from 'react';

import { Cell } from '../index';

test('Should render cell', () => {
    const cell = shallow(<Cell size={5}>test</Cell>);

    expect(cell).toMatchSnapshot();
});

test('Should render cell (offset)', () => {
    const cell = shallow(
        <Cell size={5} offset={10}>
            test
        </Cell>
    );

    expect(cell).toMatchSnapshot();
});
