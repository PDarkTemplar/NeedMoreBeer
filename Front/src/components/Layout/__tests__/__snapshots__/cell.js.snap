// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render cell (offset) 1`] = `
<div
  className="col_5 colOffset__5"
>
  test
</div>
`;

exports[`Should render cell 1`] = `
<div
  className="col_5"
>
  test
</div>
`;
